ymer/src/main/java/com/avanza/ymer/YmerFactory.java

```diff
@@ -80,12 +81,19 @@ public YmerFactory(Supplier<MongoDatabase> mongoDatabaseSupplier,
 		}
 	}
 
+	@Deprecated
```

Comment:
Suggestion to add javadoc with reference to the constructor that should be used instead, including deprecated javadoc tag

Reply:
I have added javadoc in https://github.com/AvanzaBank/ymer/pull/77/commits/fbd3279a2494975b27a976173825719ee657fb12

But maybe we should just remove this method? Any code that uses this should have a `MongoDatabaseFactory` rather than a `MongoDbFactory` anyway.